src/toolbox/useTranslation.js

```diff
@@ -23,6 +28,12 @@ const LOCALE_EN = {
     all_words: ", ",
     two_words: " and ",
     last_word: ", and "
```

Comment:
having done translations in the past, my gut tells me that the comma will never change for different language, and the connector will always just be the word "and" (so really it's just a single translation for that word).

we've written many functions in the past to "listify" an array of strings, adding commas when there are more than 2 and adding the word "and" before the last entry: https://github.com/Datawheel/canon/blob/master/packages/cms/src/utils/formatters/list.js